resources_pages/install_ds_stack_windows.md

```diff
@@ -3,290 +3,632 @@ layout: page
 
-Go to <https://git-scm.com/download/win> and a download should automatically start (if not, follow the instructions on that page). After the download has finished, run the installer selecting the following options:
-  - On the **Select Components** page, check "On the Desktop" under "Additional icons"
-  - On the **Choosing the default editor used by Git** page, select "Use the Nano editor by default" from the drop-down menu
-  - For all other pages, use the default options
+![](/resources_pages/imgs/vscode-as-git-editor.png)
 
-After installation, test if you were successful by opening the Git Bash program (if you followed the install instructions above, you should now have an icon on your Desktop). Below is a picture of the Git Bash icon on the Desktop and an opened instance of the Git Bash program:
+> Note if you wish to pin Git Bash to the taskbar, you need to search for the program in the start menu, right click the entry and select "Pin to taskbar". If you instead first launch the program and pin it by right clicking on the taskbar icon, Git Bash will open with the wrong home directory (`/` instead of `/c/users/$USERNAME`.
+
+After installation, test if you were successful by opening the Git Bash program. Below is a picture of the Git Bash icon on the Desktop and an opened instance of the Git Bash terminal (we will often refer to this as just "the terminal"):
 
 ![](/resources_pages/imgs/gitbash.png)
 
-## Python
+In the terminal, type the following to check which version of Bash you just installed:
+
+```
+bash --version
+```
+
+The output should look similar to this:
+
+```
+GNU bash, version 4.4.23(1)-release (x86_64-pc-sys)
+Copyright (C) 2019 Free Software Foundation, Inc.
+License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
+
+This is free software; you are free to change and redistribute it.
+There is NO WARRANTY, to the extent permitted by law.
+```
 
-We will be using Python for a large part of the program, including many popular 3rd party Python libraries for scientific computing. [__Anaconda__](https://www.anaconda.com) is an easy-to-install distribution of Python and most of these libraries (as well as Jupyter notebooks, one of the developing environments we will be using). We __strongly recommend__ that you use Anaconda for this program. If you insist on using your own Python setup instead of Anaconda, we will not be able to provide technical support with installation or later issues. For this program we are using __Python 3__ , not __Python 2__, so please choose the Anaconda versions that include Python 3.7
+> If you tried to paste the above into the Git Bash terminal, you will have noticed that `Ctrl+V` does not work in Git Bash. Instead you need to right click and select "Paste" or use the `Shift+Insert` shortcut. To copy from the Git Bash terminal you simply select the text you want and it is copied automatically.
```

Comment:
@ttimbers Could you confirm that this is the default setting for you? I think it is, but I am not 100% on whether I did change it and there is no "reset to default settings". I could also reinstall git bash and check later.

Reply:
I can't confirm as my Windows machine is at work and I can't get to it for another day or two. But I do remember students doing this, so let's go with it.